lib/px/utils/pxclient.lua

```diff
@@ -270,7 +270,6 @@ function M.load(px_config)
             return default_response(default_content_type, default_content)
         end
 
-        local reverse_prefix = string.sub(px_config.px_appId, 3, string.len(px_config.px_appId))
```

Comment:
this variable is used in the next line. why are you deleting it?

Reply:
you get it as a parameter from the function